source/common/unified_memory_allocation/src/memory_tracker.cpp

```diff
@@ -90,53 +90,50 @@ void *umaMemoryTrackerGetPool(uma_memory_tracker_handle_t hTracker,
     return hTracker->find(ptr);
 }
 
-struct uma_tracking_memory_provider_t {
+struct uma_memory_provider_t {
```

Comment:
I did it mostly to avoid casting the native_handle to the struct every time. With this name, it's not necessary since we have a typedef: `typedef struct uma_memory_provider_handle_t_ *uma_memory_provider_handle_t;`